src/plugins/dashboard/common/dashboard_saved_object/persistable_state/dashboard_saved_object_references.test.ts

```diff
@@ -51,190 +51,6 @@ const commonAttributes: DashboardAttributes = {
   title: '',
 };
 
-describe('legacy extract references', () => {
```

Comment:
All of these tests & the legacy extract references function have been removed because they are no longer used. It is not possible to upgrade to 8.10 without first upgrading to 7.17, and URLs from before 7.3 are unsupported as of 8.7.0 with [this PR](https://github.com/elastic/kibana/pull/144332)

Reply:
7.3 used JSON exporting which is incompatible with versions over 8.0 which require NDJSON. 7.17 was the latest version that supported JSON, so the user would have to import them to 7.17 first which would run all the required migrations.